scss/components/_top-bar.scss

```diff
@@ -155,8 +155,8 @@ $topbar-unstack-breakpoint: medium !default;
   @else {
     .top-bar-title {
       float: left;
-      margin-right: $topbar-title-spacing;
-
+      margin: $topbar-title-spacing;
+      display:inline-block;
```

Comment:
@mattferderer Currently, there is no standard for properties order in Foundation. I'm trying to improve the code standard of the framework and your PR make me think of *properties order*.

I prefer ordering CSS properties by *type* instead of alphabetically because it is easier to:
- Find a property. You know the property you are searching better by what it does (approximately) than its name.
- Understand the global component behaviour. You see and distinguish immediately the different "groups" of properties, and each property is next to those related to it.
- Understand the consequences of each property on others. Properties are ordered from the *nature/inside* to the *appearance/outside* of the component, the first having consequences on the seconds.

You can see also: https://css-tricks.com/poll-results-how-do-you-order-your-css-properties/